example/rater/huggingface_classification.ipynb

```diff
@@ -0,0 +1,481 @@
+      "Requirement already satisfied: nvidia-cufft-cu12==11.0.2.54 in /home/zayn/miniconda3/envs/uniflow/lib/python3.10/site-packages (from torch>=1.10.0->accelerate) (11.0.2.54)\n",
+      "Requirement already satisfied: nvidia-curand-cu12==10.3.2.106 in /home/zayn/miniconda3/envs/uniflow/lib/python3.10/site-packages (from torch>=1.10.0->accelerate) (10.3.2.106)\n",
+      "Requirement already satisfied: nvidia-cusolver-cu12==11.4.5.107 in /home/zayn/miniconda3/envs/uniflow/lib/python3.10/site-packages (from torch>=1.10.0->accelerate) (11.4.5.107)\n",
+      "Requirement already satisfied: nvidia-cusparse-cu12==12.1.0.106 in /home/zayn/miniconda3/envs/uniflow/lib/python3.10/site-packages (from torch>=1.10.0->accelerate) (12.1.0.106)\n",
+      "Requirement already satisfied: nvidia-nccl-cu12==2.18.1 in /home/zayn/miniconda3/envs/uniflow/lib/python3.10/site-packages (from torch>=1.10.0->accelerate) (2.18.1)\n",
+      "Requirement already satisfied: nvidia-nvtx-cu12==12.1.105 in /home/zayn/miniconda3/envs/uniflow/lib/python3.10/site-packages (from torch>=1.10.0->accelerate) (12.1.105)\n",
+      "Requirement already satisfied: triton==2.1.0 in /home/zayn/miniconda3/envs/uniflow/lib/python3.10/site-packages (from torch>=1.10.0->accelerate) (2.1.0)\n",
+      "Requirement already satisfied: nvidia-nvjitlink-cu12 in /home/zayn/miniconda3/envs/uniflow/lib/python3.10/site-packages (from nvidia-cusolver-cu12==11.4.5.107->torch>=1.10.0->accelerate) (12.3.101)\n",
+      "Requirement already satisfied: charset-normalizer<4,>=2 in /home/zayn/miniconda3/envs/uniflow/lib/python3.10/site-packages (from requests->transformers) (3.3.2)\n",
+      "Requirement already satisfied: idna<4,>=2.5 in /home/zayn/miniconda3/envs/uniflow/lib/python3.10/site-packages (from requests->transformers) (3.6)\n",
+      "Requirement already satisfied: urllib3<3,>=1.21.1 in /home/zayn/miniconda3/envs/uniflow/lib/python3.10/site-packages (from requests->transformers) (2.1.0)\n",
+      "Requirement already satisfied: certifi>=2017.4.17 in /home/zayn/miniconda3/envs/uniflow/lib/python3.10/site-packages (from requests->transformers) (2023.11.17)\n",
+      "Requirement already satisfied: MarkupSafe>=2.0 in /home/zayn/miniconda3/envs/uniflow/lib/python3.10/site-packages (from jinja2->torch>=1.10.0->accelerate) (2.1.3)\n",
+      "Requirement already satisfied: mpmath>=0.19 in /home/zayn/miniconda3/envs/uniflow/lib/python3.10/site-packages (from sympy->torch>=1.10.0->accelerate) (1.3.0)\n"
+     ]
+    }
+   ],
+   "source": [
+    "%reload_ext autoreload\n",
+    "%autoreload 2\n",
+    "\n",
+    "import sys\n",
+    "\n",
+    "sys.path.append(\".\")\n",
+    "sys.path.append(\"..\")\n",
+    "sys.path.append(\"../..\")\n",
+    "\n",
+    "!{sys.executable} -m pip install transformers accelerate bitsandbytes scipy"
```

Comment:
let's do `!{sys.executable} -m pip install -q transformers accelerate bitsandbytes scipy` for quite install without verbose logs.